plane/source/docs/aborting-autolanding.rst

```diff
@@ -0,0 +1,45 @@
+.. _aborting-autolanding:
+
+===========================
+How to Abort an Autolanding
+===========================
+A landing-abort mechanism is provided to allow you to abort a landing sequence in a safe, controlled, and expected way. Custom abort behaviour can be pre-programmed as part of the mission or you can use the default abort mechanism. To enable this feature set param LAND_ABORT_THR=1.
```

Comment:
Probably good to make LAND_ABORT_THR a ref: ?

Reply:
yes...missed that one